integrations/snyk/snyk/client.py

```diff
@@ -111,7 +123,11 @@ async def get_issues(self, org_id: str, project_id: str) -> list[dict[str, Any]]
                 method="POST",
                 version=self.snyk_api_version,
             )
-        )["issues"]
+        ).get("issues")
+
+        if not issues:
+            return []
```

Comment:
```suggestion
        ).get("issues", [])
```

Reply:
I don't want [] to be cached so i return it immediately